sirepo/pkcli/job_cmd.py

```diff
@@ -117,6 +117,9 @@ def _do_fastcgi(msg, template):
             r = b''
             while True:
                 r += s.recv(int(1e8))
+                if not r:
+                    pkdlog('socket closed abruptly')
```

Comment:
log what agent, etc.

I can't add a comment to L134. Like we do in job_supervisor, we should have a max retries (or exceptions) in this loop. If there are too many retries, the job_cmd should exit with logging info. The job_agent should be prepared that the job_cmd exits prematurely -- it probably already handles this case.

If the job_agent finds itself looping on premature job_cmd exits, then it should exit or at least log the error back to the supervisor.

Reply:
The job_agent will [log information](https://github.com/radiasoft/sirepo/blob/8860e72cf5ea17fd82f67e1a8b2a2bd57a12e995/sirepo/pkcli/job_agent.py#L851) about the agent.

I've added the retries. All errors make it back to the supervisor.